tanner/emulators/mysqli.py

```diff
@@ -6,7 +6,7 @@ def __init__(self, db_name):
         self.db_name = db_name
         self.helper = mysql_db_helper.MySQLDBHelper()
 
-    async def setup_db(self, query_map):
+    async def setup_db(self):
```

Comment:
Now you need to correct this where this method is called, so maybe do this in separate PR.

Reply:
This is not called anywhere else apart from the tests.